daemon/sdnotify.go

```diff
@@ -55,9 +60,5 @@ func SdNotify(unsetEnvironment bool, state string) (sent bool, err error) {
 	defer conn.Close()
 
 	_, err = conn.Write([]byte(state))
-	// Error sending the message
-	if err != nil {
-		return false, err
-	}
-	return true, nil
+	return err == nil, err
```

Comment:
I would keep logic outside of return statement.
Even if I agree the original form may be overly verbose, I think it is the expected idiomatic form.